src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,624 @@
+import {
+  PrimaryAddressViewField,
+  getPage
+} from '../helpers';
+
+import initialData from '../../../../../test/disability-benefits/526EZ/schema/initialData';
+
+function isValidZIP(value) {
+  if (value !== null) {
+    return /^\d{5}(?:(?:[-\s])?\d{4})?$/.test(value);
+  }
+  return true;
+}
+
+function isValidPhone(value) {
+  if (value !== null) {
+    return /^\d{10}$/.test(value) || /^\d{11}$/.test(value);
+  }
+  return true;
+}
+
+function validatePhone(errors, phone) {
+  if (phone && !isValidPhone(phone)) {
+    errors.addError(
+      'Phone numbers must be at least 10 digits (dashes allowed)'
+    );
+  }
+}
+
+function validateZIP(errors, zip) {
+  if (zip && !isValidZIP(zip)) {
+    errors.addError('Please enter a valid 5 or 9 digit ZIP (dashes allowed)');
+  }
+}
+
+const typeLabels = {
+  DOMESTIC: 'Domestic',
+  MILITARY: 'Military',
+  INTERNATIONAL: 'International'
+};
+
+const states = [
+  'AL',
+  'AK',
+  'AS',
+  'AZ',
+  'AR',
+  'AA',
+  'AE',
+  'AP',
+  'CA',
+  'CO',
+  'CT',
+  'DE',
+  'DC',
+  'FM',
+  'FL',
+  'GA',
+  'GU',
+  'HI',
+  'ID',
+  'IL',
+  'IN',
+  'IA',
+  'KS',
+  'KY',
+  'LA',
+  'ME',
+  'MH',
+  'MD',
+  'MA',
+  'MI',
+  'MN',
+  'MS',
+  'MO',
+  'MT',
+  'NE',
+  'NV',
+  'NH',
+  'NJ',
+  'NM',
+  'NY',
+  'NC',
+  'ND',
+  'MP',
+  'OH',
+  'OK',
+  'OR',
+  'PW',
+  'PA',
+  'PR',
+  'RI',
+  'SC',
+  'SD',
+  'TN',
+  'TX',
+  'UT',
+  'VT',
+  'VI',
+  'VA',
+  'WA',
+  'WV',
+  'WI',
+  'WY',
+  'PI',
+  'UM'
+];
+
+const stateLabels = {
+  AL: 'Alabama',
+  AK: 'Alaska',
+  AS: 'American Samoa',
+  AZ: 'Arizona',
+  AR: 'Arkansas',
+  AA: 'Armed Forces Americas (AA)',
+  AE: 'Armed Forces Europe (AE)',
+  AP: 'Armed Forces Pacific (AP)',
+  CA: 'California',
+  CO: 'Colorado',
+  CT: 'Connecticut',
+  DE: 'Delaware',
+  DC: 'District Of Columbia',
+  FM: 'Federated States Of Micronesia',
+  FL: 'Florida',
+  GA: 'Georgia',
+  GU: 'Guam',
+  HI: 'Hawaii',
+  ID: 'Idaho',
+  IL: 'Illinois',
+  IN: 'Indiana',
+  IA: 'Iowa',
+  KS: 'Kansas',
+  KY: 'Kentucky',
+  LA: 'Louisiana',
+  ME: 'Maine',
+  MH: 'Marshall Islands',
+  MD: 'Maryland',
+  MA: 'Massachusetts',
+  MI: 'Michigan',
+  MN: 'Minnesota',
+  MS: 'Mississippi',
+  MO: 'Missouri',
+  MT: 'Montana',
+  NE: 'Nebraska',
+  NV: 'Nevada',
+  NH: 'New Hampshire',
+  NJ: 'New Jersey',
+  NM: 'New Mexico',
+  NY: 'New York',
+  NC: 'North Carolina',
+  ND: 'North Dakota',
+  MP: 'Northern Mariana Islands',
+  OH: 'Ohio',
+  OK: 'Oklahoma',
+  OR: 'Oregon',
+  PW: 'Palau',
+  PA: 'Pennsylvania',
+  PR: 'Puerto Rico',
+  RI: 'Rhode Island',
+  SC: 'South Carolina',
+  SD: 'South Dakota',
+  TN: 'Tennessee',
+  TX: 'Texas',
+  UT: 'Utah',
+  VT: 'Vermont',
+  VI: 'Virgin Islands',
+  VA: 'Virginia',
+  WA: 'Washington',
+  WV: 'West Virginia',
+  WI: 'Wisconsin',
+  WY: 'Wyoming',
+  PI: 'Rizal state',
+  UM: 'United States Minor Outlying Islands'
+};
+
+const countries = [
+  'USA',
```

Comment:
@markgreenburg suggested this should be provided with a label so that it is displayed as "United States of America", however, I couldn't find other instances in which we use that name in other forms, @peggygannon is this consistent with the content strategy?

Reply:
This will be changed to "United States" as that is used elsewhere.